turbo/jsonrpc/eth_call.go

```diff
@@ -464,6 +464,55 @@ func verifyExecResult(execResult *core.EphemeralExecResult, block *types.Block)
 	return nil
 }
 
+// verify the state consistency between the history state reader and the hph.Ctx reader
+func verifyStateConsistency(touchedPlainKeys [][]byte, stateReader state.StateReader, hph *commitment.HexPatriciaHashed) (bool, error) {
+	ctx := hph.GetPatriciaContext()
+	for _, key := range touchedPlainKeys {
+		if len(key) == 20 { //account address
```

Comment:
Should throw err if `len<20`? Or is this impossible?

Reply:
No, that will never happen. The keys will either be account keys (20 bytes), or storage keys (20+32=52 bytes).

Reply:
that would happen during tests at least